doc/source/reference/commands/dump.rst

```diff
@@ -195,6 +223,16 @@ To exclude indexes from the output, specify ``no``.
 
 The default value is ``yes``.
 
+``sort_hash_table``
+"""""""""""""""""""
+
+.. versionadded:: 7.0.5
+
+You can ascending sort the output of hash table when it contains hash table.
+To don't sort the output of hash table, specify ``no``.
+
```

Comment:
It seems that sort by `_key` is important information. but missing from above sentence.